### PR TITLE
convert itemLocked to use modal component

### PR DESCRIPTION
**Pull request by d88naimi**
Converting itemLocked to use design systems modal compoenent

### CODE REVIEW

**Comment by d88naimi** [comment]
Screen Shot 2021-07-29 at 10 19 14 AM

Screen Shot 2021-07-29 at 10 20 36 AM

Screen Shot 2021-07-29 at 10 20 44 AM

**Comment by d88naimi** [comment]
@shrunyan Updated version 2.0: Swapped out modal component to confirmDialog, verified that confirmDialog renders in `npm run stage:start` 
tested with the same item edit in one browser and the other in incognito browser.

File: src/apps/content-editor/src/app/components/LockedItem/LockedItem.less [hunk 1]
```diff
@@ -1,58 +1,42 @@
...
-    border: 1px solid @zesty-orange;
-    max-width: 550px;
-    margin-left: -15%;
+.ItemLocked {
+  .ConfirmDialog {
+    background: linear-gradient(@zesty-orange 35%, @appBkgColor 25%);
+    position: relative;
+    overflow: hidden;
+
+    .backgroundIcon {
```
Comment (d88naimi) [inline src/apps/content-editor/src/app/components/LockedItem/LockedItem.less:10]: @shrunyan 

Can you point me in the right direction im struggling to get the svg icon to overflow:hidden 
as you can see in the image the svg is overlapping some font resulting in a lighter font color. 

Screen Shot 2021-07-29 at 4 57 59 PM

**Comment by d88naimi** [comment]
@shrunyan version 3.0 updated


Screen Shot 2021-07-30 at 8 52 36 AM

File: src/apps/content-editor/src/app/components/LockedItem/LockedItem.less (hunk 1 above)
Comment (d88naimi) [reply src/apps/content-editor/src/app/components/LockedItem/LockedItem.less:10]: Resolved: Added position relative on `ConfirmContent` allowed the content to be on top of svg preventing grayed out text .

**Review by shrunyan — APPROVED** [review]

**Pull request merged**